test cases/format/4 config/meson.build

```diff
@@ -14,6 +15,6 @@ foreach name, f : meson_files
   test(
     name,
     meson_cmd,
-    args : [ 'format', '--check-only', f ],
+    args : [ 'format', '--check-only', f, '--configuration', meson_format ],
```

Comment:
You should not add path to meson.format here, as the goal of this test is to ensure meson.format is automatically found.